File: app/src/main/java/com/seafile/seadroid2/framework/model/dirents/DeleteDirentModel.java
package com.seafile.seadroid2.framework.model.dirents;

import com.seafile.seadroid2.framework.model.ResultModel;

public class DeleteDirentModel extends ResultModel {
    public String commit_id;

    @Override
    public String toString() {
        return "DeleteDirentModel{" +
                "commit_id='" + commit_id + '\'' +
                ", success=" + success +
                ", error_msg='" + error_msg + '\'' +
                '}';
    }
}

File: app/src/main/java/com/seafile/seadroid2/framework/model/dirents/DeleteDirentModel.kt
package com.seafile.seadroid2.framework.model.dirents

import com.seafile.seadroid2.framework.model.ResultModel

class DeleteDirentModel : ResultModel() {
    @JvmField
    var commit_id: String? = null

    override fun toString(): String {
        return "DeleteDirentModel(commit_id=$commit_id, success=$success, error_msg=$error_msg)"
    }
}
